portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -151,6 +137,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     return blogMessages;
   }
 
+  private List<String> updateTagsToSearch (List<String> tagsToSearch) {
```

Comment:
This function could probably have a better name like getUserFollowedTags.
Or some comments about what it does.

Reply:
Another thing: why does this method need to return a List<String> instead of a "void"? I think it would be confusing what the return value is for. It is usually common to return void and directly update what is referenced by the argument.